kahuna/public/js/main.js

```diff
@@ -461,8 +461,17 @@ kahuna.directive('uiTitle', ['$rootScope', function($rootScope) {
                   var title = (titleFunc ? titleFunc(toParams) : toState.name) +
                           ' | ' + attrs.uiTitleSuffix;
 
+                  attrs.uiTitle = title;
+
                   element.text(title);
             });
+
+            $rootScope.$on('events:new-images', 
+                function(event, data) {
+                    var title = (data.count ? '(' + data.count + ' new) ' + attrs.uiTitle : attrs.uiTitle);
```

Comment:
We can use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here to improve the readability of the ternary a little:

```suggestion
                    var title = (data.count ? `(${data.count} new)  ${attrs.uiTitle}` : attrs.uiTitle);
```

Reply:
Thanks for the suggestion on improving readability.